### PR TITLE
MICS-DEV Update dependencies and logging

### PR DESCRIPTION
**Pull request by nbelie**
- Change formatting of logs in order not to have a JSON but a proper line
- Update packages version part 1 - Especially handlebars that has security issues

### CODE REVIEW

**Comment by PoulpiFr** [comment]
https://docs.google.com/document/d/1BKV_oqmIFDwWZp3w-Ah3uz66keDYF0x0MNeHQn3lh1E/edit

So:
- you shouldn't be the one having to update the SDK version -> did you do it for a special reason, like for testing?
- Can you add an entry in the changelog?

File: examples/activity-analyzer/src/tests/index.ts [hunk 1]
```diff
@@ -23,6 +24,7 @@ describe("Test Example Activity Analyzer", function () {
     };
 
     const itFactory = helpers.itFactory(new MyActivityAnalyzerPlugin(), activityAnalyzerProperties);
+    helpers.mockApi(/\/v1\/activity_analyzers\/(.){1,10}\/properties/).returns(activityAnalyzerProperties);
```
Comment (PoulpiFr) [inline examples/activity-analyzer/src/tests/index.ts]: Why did you add that?

If it is required for the tests to pass, how could the code without this line land on master previously?

**Comment by nbelie** [comment]
If not we who ?
- It's been a few weeks that there are security issues on dependencies. 
- One of the previous update was mis-formatting the logs (maybe mine)
- I'm using it everyday, just trying to improve it to improve my day to day

File: examples/activity-analyzer/src/tests/index.ts (hunk 1 above)
Comment (nbelie) [reply examples/activity-analyzer/src/tests/index.ts]: Indeed this one is a mistake :) I'll update the PR

File: src/mediarithmics/plugins/common/BasePlugin.ts [hunk 2]
```diff
@@ -533,8 +533,12 @@ export abstract class BasePlugin {
 
     this.app.use(bodyParser.json({type: "*/*", limit: "5mb"}));
     
-    this.logger = winston.createLogger();
-    this.logger.add(new winston.transports.Console());
+    this.logger = winston.createLogger({
```
Comment (PoulpiFr) [inline src/mediarithmics/plugins/common/BasePlugin.ts]: So, apparently we broke the ability to log JSON without having to do explicit JSON.stringify (which is quite CPU intensive) when we upgraded winston to the 3.x release.

We should add the format.splat() to fix that.

See:
```
const winston = require("winston");

const object = {hello: "world"};

const brokenLog = winston.createLogger({
    format: winston.format.simple(),
    transports: [
        new winston.transports.Console()
    ]
});

brokenLog.info("[broken] Simple test %j", object);

const log = winston.createLogger({
    format: winston.format.combine(
        winston.format.splat(),
        winston.format.simple()
    ),
    transports: [
        new winston.transports.Console()
    ]
});

log.info("Simple test %j", object);
```

returns:
```
info: [broken] Simple test %j
info: Simple test {"hello":"world"}
```

File: examples/activity-analyzer/src/tests/index.ts (hunk 1 above)
Comment (PoulpiFr) [reply examples/activity-analyzer/src/tests/index.ts]: Did you? =)

**Comment by PoulpiFr** [comment]
Aside from my last comment, LGTM!

File: examples/activity-analyzer/src/tests/index.ts (hunk 1 above)
Comment (nbelie) [reply examples/activity-analyzer/src/tests/index.ts]: Sorry about that -> done
